addons/html_editor/static/tests/insert/html.test.js

```diff
@@ -211,7 +218,7 @@ describe("collapsed selection", () => {
             )
         );
         expect(getContent(editor.editable)).toBe(
-            `<p>cont</p><p class="oe_unbreakable">1</p><p class="oe_unbreakable">2[]</p><p><br></p>`
+            `<p>cont</p><p class="oe_unbreakable">1</p><p class="oe_unbreakable">2[]</p>`
```

Comment:
I guess this test, and maybe others too, don't make sense anymore and should be removed.

Reply:
I don't know, it shows that the placeholders are inserted where we need them though.